server/models/ProfileModel.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  userId: {
+    type: String,
+    required: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  register_date: {
+    type: Date,
+    default: Date.now,
+  },
+  admin: {
+    type: Boolean,
+    default: false,
+  },
+  address: String,
+  city: String,
+  phone: String,
+  profileImage: String,
+  photos: [],
+  jop: String,
+  description: String,
+  birth: String,
+  gender: String,
+  availability: Boolean,
```

Comment:
Will this design for availability fit our use cases? How will the sitter be able to pick the days of the week on which they choose to be available for service? :)

Reply:
Can it be like   {availability : [Date] }. ?
Array of dates

Reply:
To keep things simple for now - availability will be the days of the week (Sun-Sat), the sitter can chose to work on certain days of the week. For example, Monday, Tuesday and Thursday.

Eventually, we can add a time period as well (start - end). For example, (2021/09/20 - 2021/12/31)